srcpkgs/python3-blinker/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'python3-blinker'
 pkgname=python3-blinker
-version=1.4
-revision=7
+version=1.5
+revision=1
 wrksrc="blinker-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 makedepends="python3-devel"
 depends="python3"
+checkdepends="python3-pytest"
 short_desc="Fast, simple object-to-object and broadcast signaling for Python3"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
-homepage="http://pythonhosted.org/blinker/"
+homepage="https://blinker.readthedocs.io/en/stable/"
```

Comment:
```suggestion
homepage="https://blinker.readthedocs.io/en/stable/"
changelog="https://raw.githubusercontent.com/pallets-eco/blinker/main/CHANGES.rst"
```

Reply:
Should this be a link to the raw file? Perhaps `https://blinker.readthedocs.io/en/stable/#changes` would be better?

Reply:
yes, the raw file is best so it's easy to view in a terminal

Reply:
Good point: I should've checked Manual.md before asking.